recipes/genpy/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "genpy" %}
+{% set version = "2016.1.3" %}
+{% set sha256 = "1c11726f1e8ace8bbdfc87816403c9a59f53a8c3d45c99187ae17c9725d87a91" %}
+
+package:
+    name: {{ name|lower }}
+    version: {{ version }}
+
+source:
+    fn: {{ name }}-{{ version }}.tar.gz
+    url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+    build:
+        - python
+        - setuptools
+        - six
+        - numpy
+        - pytools
+    run:
+        - python
+        - numpy
+        - pytools
+        - six
+
+test:
+    imports:
+        - {{ name }}
+
+about:
+    home: http://pypi.python.org/pypi/{{ name }}
```

Comment:
@inducer I merged this but it would be really helpful if you could change `{{ name }}` to `genpy` in the feedstock once it is created.